report.Rmd

```diff
@@ -10,6 +10,7 @@ params:
     language links that are similar to the individual search results. A
     control group will see the currently existing search results page.
   data: "./data/Explore similar pages, categories and suggested languages in search results/events_raw_20170724112938.RData"
+  query: "SELECT * FROM TestSearchSatisfaction2_16909631;"
```

Comment:
This will override the query created from the other params and the report will end up fetching the whole table. Maybe `query: NULL` or something?